docs/en/transform-v2/explode.md

```diff
@@ -0,0 +1,67 @@
+# Explode
+
+> Explode transform plugin
+
+## Description
+
+Split a row of data into multiple data according to the specified delimiter.
```

Comment:
Hi, this code basic is looks good to me.

But I have a thoughts about this explode feature, can we only support explode a list type column? the string split phase move to another phase, like add a sql function, `split(str_col, ';')`


More over, we can add `explode` as a sql function, then the transform all use `Sql` transfrom. the sql is looks like 
`select explode(split(str_col,';')) from `

(Just discussion, WDYT @Hisoka-X )

Reply:
Thanks @liunaijie , I think you speak to the point.
> More over, we can add explode as a sql function

Yes, I like sql function more.

> select explode(split(str_col,';'))

This way more clear and better. cc @CosmosNi WDYT?

Reply:
OK, I'll try.

Reply:
PS: we should add `LATERAL VIEW` keywords too. Please refer https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-lateral-view.html. It is reasonable to split it into multiple PRs

Reply:
@liunaijie  @Hisoka-X  But SQLTransform only support  one to one. So edit SQLTransform to support flatmap?

Reply:
Yep, some code in this pr can be reused.